build-system/test-configs/forbidden-terms.js

```diff
@@ -919,12 +919,15 @@ const forbiddenTermsSrcInclusive = {
       'extensions/amp-consent/0.1/consent-state-manager.js',
     ],
   },
-  '(cdn|3p)\\.ampproject\\.': {
+  '((cdn|3p)\\.ampproject\\.)|(ampproject\\.(org|net))': {
```

Comment:
Can this be replaced with just`ampproject\\.`?

Reply:
i believe it can, let me try if it catches any false positives